src/pynws/raw_data.py

```diff
@@ -92,30 +106,48 @@ async def raw_points(
 
 
 async def raw_detailed_forecast(
-    wfo: str, x: int, y: int, websession: ClientSession, userid: str
+    wfo: str,
+    x: int,
+    y: int,
+    websession: ClientSession,
+    userid: str,
+    forecast_units: str,
```

Comment:
According to https://www.weather.gov/documentation/services-web-api#/default/gridpoint, `units` is not a valid input param for this endpoint.  If correct, we should not include it for the detailed forceast.

Reply:
Missed that one. That was an initial test that I was doing!